### PR TITLE
Update torch_engine.py from "loss = loss_fn(enc_output, dec_output" to "loss = loss_fn(dec_output, target_img)"

### PR DESCRIPTION
**Pull request by AliceSum**

### CODE REVIEW

**Comment by oke-aditya** [comment]
Absolutely right @JonathanSum . I missed this thing in `val_step`. It is correct in `train_step`. 
Thnx for this !!

**Comment by oke-aditya** [comment]
The reason for such bugs was because I trained on Kaggle, where I used the `ipynb` file and not the scripts. Such stuff fixed on fly might not be there in scripts.
Thnx for that !!

**Comment by AliceSum** [comment]
sato love
